src/core/agent/services/signifyNotificationService.ts

```diff
@@ -189,6 +189,11 @@ class SignifyNotificationService extends AgentService {
         return;
       }
     }
+
+    if (notif.a.r === NotificationRoute.ExnIpexAgree) {
+      await Agent.agent.ipexCommunications.grantAcdcFromAgree(notif.a.d);
+      return;
```

Comment:
@Sotatek-BaoHoanga DTIS-1045 will change this but until that is merged I think we should call `await this.markNotification(notif.i);` here before the `return`

Reply:
Pushed